R/font.R

```diff
@@ -115,22 +115,22 @@ checkMissingType1Fonts <- function(fonts, fontTable) {
 }
 
 
-## Look up font in extrafont::fonttable and
+## Look up font in systemfonts::system_fonts() and
```

Comment:
This comment can actually remain unchanged, right?  (because we have actually stuck with extrafont::fonttable for now)

Reply:
Yes, that's correct. The original comment should be kept here.